src/components/WorkOrder/helpers.js

```diff
@@ -66,55 +67,198 @@ export function getTimeLogs(id) {
     .then(res => res.data.records);
 }
 
-export function getSignalsOptions(searchValue) {
-  return api
-    .workOrder()
-    .signals(searchValue)
-    .then(res => res.data.records);
+const formatSocrataResponseToKnackFormat = resArray =>
+  resArray.data.map(asset => ({
+    id: asset.id,
+    identifier: "📍 " + asset.location_name,
```

Comment:
Good idea and this gave me a good reason to play with BrowserStack! The emoji still renders as a red push pin on Android devices, but it is probably best to use something more predictable in the future. I'll keep this in mind for Knack Signs & Markings (since it uses this emoji too).